rsmtool/modeler.py

```diff
@@ -1060,6 +1060,7 @@ def train_skll_model(self,
                          filedir,
                          figdir,
                          file_format='csv',
+                         custom_fixed_parameters={},
```

Comment:
Nitpick: While probably not a big deal in this case, it might be better if the default argument did not have a mutable value because weird things could happen in the future if the parameter's value were to be mutated within this function. Maybe using `None` instead and then modifying where it's used to substitute in an empty dictionary if the value is `None` would be slightly more future-proof.

Reply:
Good call! I will fix this.